### DISCUSSION TITLE
Do we need a peer-review process for campaign proposals?

### DISCUSSION THREAD

**Discussion by CooperSmout** [FreeOurKnowledge/website#43] [Ideas]
**Is your feature request related to a problem? Please describe.**
The number of proposed campaigns is growing by the day, so it could be useful to formalise a peer-review process so that we can (a) strengthen proposals, and (b) choose the most appropriate campaigns to post on the website.

**Describe the solution you'd like**
Some kind of codified review process, ideally with metrics representing different qualities. At a first pass, these might be:
* Novelty. How dissimilar is the proposal to other campaigns that have been proposed/posted on FOK?
* Importance. If the campaign is successful, how beneficial will it be to the signatories and broader community?
* Community-interest. This could be as simple as counting the number of likes and comments in the Github Issue thread.
* Clarity. Is the proposed action simple and clearly specified? Is the target audience and threshold clearly specified?
* Likelihood of success. What are the chances the campaign will reach the target threshold?
* Accountability. Can the actions be monitored in some way to check compliance?

**Describe alternatives you've considered**
Alternatively, we could just post every campaign that gets proposed, see what kind of traction they get, and only leave up those that seem like they're getting community interest. One downside of this strategy is that we miss an opportunity to strengthen campaigns before posting to the website. Another is that we might overwhelm users by giving them too many options and/or dilute our 'brand' by posting poorly-conceived campaigns. I think that in the early days of the project at least, it's probably best to screen campaigns to some extent, and use the Github features to gauge community interest before posting.

**Additional context**
We'll also need to work out how to source peer-reviewers from the community and allocate them to campaigns

**Comment by dlholf** [comment]
> Alternatively, we could just post every campaign that gets proposed, see what kind of traction they get, and only leave up those that seem like they're getting community interest.

If the campaigns route through the forum first anyway, that might just act as a 'first pass' (as mentioned in the community interest metric). People would likely gravitate towards discussing the campaigns they would like to support and (hopefully) offer suggestions as part of that. 

> Some kind of codified review process, ideally with metrics representing different qualities.

Would it make sense to work it into the proposal system itself? (i.e., the proposers need to say how each of the qualities are met within the proposal)
